libs/docker-client/src/main/kotlin/batect/docker/pull/RegistryCredentialsSource.kt

```diff
@@ -90,10 +90,10 @@ data class HelperBasedCredentialsSource(val helperName: String, val serverAddres
 
         val serverUrl = parsed.getStringMember("ServerURL")
 
-        if (username == "<token>") {
```

Comment:
Ditto: this might be even clearer as a `when`.

Reply:
Please check if you agree to my rationale above.

Reply:
I actually still gave it a try, and indeed found it to result in more compact code, so done (in both places).